### PR TITLE
Privacy Policy Text edit

### PR DESCRIPTION
**Pull request by pieckenst**

### CODE REVIEW

**Comment by JDJGInc** [comment]
Problem with this is it doesn't translate my meaning correctly

**Comment by JDJGInc** [comment]
The support channel stores user id, the thread I'd, and the time it is last used(for archive reasons), it's also opt in

**Comment by pieckenst** [comment]
Ok feel free to check that support phrase again

**Comment by JDJGInc** [comment]
That one is good, there's more but this could probably be shortened

**Comment by pieckenst** [comment]
I dont exactly see what could be shortened there but ya should be able to edit it tbh

**Comment by JDJGInc** [comment]
First one is when the bot joins a guild or leaves a guild we log that in our logging channel

**Comment by JDJGInc** [comment]
> I dont exactly see what could be shortened there but ya should be able to edit it tbh

Different wording?
Tbh I couldn't shorten it either,
Second one is that your info may be in errors, but it's temporary

**Comment by JDJGInc** [comment]
Not sure if the ram warning would count?

**Comment by pieckenst** [comment]
Edited that errors phrase

**Comment by pieckenst** [comment]
> First one is when the bot joins a guild or leaves a guild we log that in our logging channel
Tbh you kinda can get that meaning from the phrase that i put there + it repeats same word less than whats currently there

**Comment by JDJGInc** [comment]
We store who uses and what they use if for example they did te*tadpole

The command tadpole and who did it would be stored

**Comment by JDJGInc** [comment]
We store user ids for economy but this being moved to my local database, so no longer mongodb(may be outdated though)

**Comment by JDJGInc** [comment]
> We store user ids for economy but this being moved to my local database, so no longer mongodb(may be outdated though)

Don't quote me on this, the user id being used for economy commands is good

**Comment by JDJGInc** [comment]
Typo of occure, you need occured .___.

**Comment by JDJGInc** [comment]
In class now will do this on PC soon

**Comment by pieckenst** [comment]
Fixed it)

**Comment by JDJGInc** [comment]
Does look better but albeit is not really a standard word, I want my text to be simple and not jargon

**Comment by JDJGInc** [comment]
I don't want to be like those damn legal documents, I want simple, concise, something a 13 year old can easily go ah okay

**Comment by JDJGInc** [comment]
And for an adult to go okay easy and concise

**Comment by JDJGInc** [comment]
But still not babyish

**Comment by JDJGInc** [comment]
Oh yeah field on we will store the guild id and channel id if you do member joins/leaves, and member changes for example status, etc.

**Comment by JDJGInc** [comment]
Statuses aren't really releveant for scanning statuses, so... 

Yeah all good

File: cogs/Bot.py [hunk 1]
```diff
@@ -237,15 +237,15 @@ async def promise(self,ctx):
...
+    embed = discord.Embed(title = "Privacy Policy", color = random.randint(0, 16777215))
+    embed.add_field(name = "1:", value = "We have a logging channel that notifies us when the bot joins or leaves a guild", inline = True)
+    embed.add_field(name = "2:", value = "We log errors that occured in the bot(although it might contain private information). This is only visible temporarily in console, and is not stored in any of our DBs.")
+    embed.add_field(name = "3:", value = "We store user ids for economy commands (Opt In)", inline = True)
+    embed.add_field(name = "4:", value = "We store user id , and support channel id, as well as last time it was used(for archive reasons), for allowing ticket based support. (Opt In)", inline = True)
+    embed.add_field(name = "5:", value = "We store inputted invalid commands", inline = True)
+    embed.add_field(name = "6:", value = "We may temporarily look at your mutual guilds with our bot or the list of servers our bot is in, with some information, just to check if any problems occure(like if a command is going hayware) or to prevent abuse. If you want to look at what is sent in embeds, just ask, We will show you.", inline = True)
```
Comment (JDJGInc) [inline cogs/Bot.py]: error here with occured, it should be occured, but occure is what it says.

**Comment by pieckenst** [comment]
> Oh yeah field on we will store the guild id and channel id if you do member joins/leaves, and member changes for example status, etc.

Dont really know where to put this so feel free to edit it in )

**Comment by JDJGInc** [comment]
> ut this so feel free to edit it in )

You didn't make it possible to edit your pull request :(

**Comment by JDJGInc** [comment]
Put it at as rule 8, so after rule 8, but before final, also

**Comment by JDJGInc** [comment]
image
This so, yeah there's another to get the new privacy policy to be done.

**Comment by pieckenst** [comment]
Does that sound good?

**Comment by JDJGInc** [comment]
> Does that sound good?

Once you update the two fields I commented on about, I believe it should be good to go.

**Comment by pieckenst** [comment]
> At least to merge in, if I still don't like it, I can just ask you to make a new pull request for it.

I updated the economy field thing already , and where you didnt like albeit replaced it with alhough , and well the logging channel thing is the best i could come up with

**Comment by JDJGInc** [comment]
> > At least to merge in, if I still don't like it, I can just ask you to make a new pull request for it.
> 
> I updated the economy field thing already , and where you didnt like albeit replaced it with alhough , and well the logging channel thing is the best i could come up with

just read the two comments above, and once you fixed them, tell me and I can review them(they are literately called comments)

**Comment by JDJGInc** [comment]
I may also make a deal with you, if you can follow it to be an owner on the bot as well, but that will be DM only lol

**Review by JDJGInc — CHANGES_REQUESTED** [review]

File: cogs/Bot.py (hunk 1 above)
Comment (pieckenst) [reply cogs/Bot.py]: I fixed that https://github.com/JDJGInc/JDBot/pull/9/commits/315b0782126ac36fe934ac3ba2d7132710d58651 here

Comment (JDJGInc) [reply cogs/Bot.py]: alrght, time to merge and test it.

Comment (pieckenst) [reply cogs/Bot.py]: So i dunno how you didnt notice that

Comment (JDJGInc) [reply cogs/Bot.py]: Should be all good to @pieckenst so yeah :D

**Pull request merged**